cpp/src/TrinoBridge.cpp

```diff
@@ -250,6 +253,11 @@ class JniHandle {
                                         fragment.planNode->toString(true, true));
 
       auto task = exec::Task::create(id.fullId(), std::move(fragment), id.id(), queryCtx);
+      if (!nativeConfigs_->getSpillDir().empty()) {
+        std::string fullPath = nativeConfigs_->getSpillDir() + "/spill-" + id.fullId();
+        std::filesystem::create_directory(fullPath);
```

Comment:
I am not sure if the parent directory doesn't exist, what will happen here?

Reply:
this API will return `false` and will spill will not enable. refactored this part.

Reply:
I think maybe we could simply use `std::filesystem::create_directories`, refer to [create_directory](https://en.cppreference.com/w/cpp/filesystem/create_directory), it seems `std::filesystem::create_directories` could create full path no matter whether parent directory exists or not.

Reply:
nice catch, updated.